src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -63,12 +57,12 @@ const getInstallation = async (client: GitHubAPI, subscription: Subscription, lo
 
 	} catch (err) {
 		log.error(
-			{ installationId: gitHubInstallationId, error: err, uninstalled: err.code === 404 },
+			{ installationId: gitHubInstallationId, error: err, uninstalled: err.status === 404 },
 			"Failed connection"
 		);
 		statsd.increment(metricError.failedConnection);
 
-		return Promise.reject({ error: err, id: gitHubInstallationId, deleted: err.code === 404 });
+		return Promise.reject({ error: err, id: gitHubInstallationId, deleted: err.status === 404 });
```

Comment:
is it from code to status correct?  Fairly certain this is an error of type `AxiosError` and the correct check is `err.response.status`

Reply:
investigating

Reply:
Good news everyone.
Throws `AxiosError` with `err.response.status` but then gets intercepted and becomes a `GithubClientError` before it reaches the catch block.... at this point it becomes err.status.

We will get failed tests if this changes :)

Reply:
ah right.  In that case, maybe change the error type? :)

Reply:
It's a try/catch error so typescript wont allow me to type it as anything other than any or undefined.
Unless I misunderstood?

Reply:
ah right, stupid try/catch